packages/react-native-renderer/src/__tests__/ReactFabric-test.internal.js

```diff
@@ -30,8 +32,16 @@ describe('ReactFabric', () => {
     ReactFabric = require('react-native-renderer/fabric');
     FabricUIManager = require('FabricUIManager');
     UIManager = require('UIManager');
+    createReactClass = require('create-react-class/factory')(
```

Comment:
I copied this from https://github.com/facebook/react/blob/b87aabdfe1b7461e7331abb3601d9e6bb27544bc/packages/react/src/__tests__/createReactClassIntegration-test.internal.js#L24-L27